### PR TITLE
Encounters completion stats - Nutrition group 

### PR DESCRIPTION
**Pull request by anvmn**
#1285

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_810

**Comment by anvmn** [comment]
@AronNovak  please review. Docs are added at #1288.

File: server/hedley/modules/custom/hedley_reports/scripts/generate-nutrition-group-completion-data.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    // No more items left.
+    break;
+  }
+
+  $ids = array_keys($result['node']);
+  $nodes = node_load_multiple($ids);
+  foreach ($nodes as $node) {
+    $completion_data = hedley_reports_generate_completion_data_for_nutrition_group_encounter($node);
+    $node->field_reports_data[LANGUAGE_NONE][0]['value'] = json_encode($completion_data);
+    node_save($node);
```
Comment (AronNovak) [inline server/hedley/modules/custom/hedley_reports/scripts/generate-nutrition-group-completion-data.php:72]: try-catch, error handling perhaps?

Comment (anvmn) [reply server/hedley/modules/custom/hedley_reports/scripts/generate-nutrition-group-completion-data.php:72]: @AronNovak 

Similar to https://github.com/TIP-Global-Health/eheza-app/pull/1283#discussion_r1730425233

**Review by AronNovak — APPROVED** [review]

**Pull request merged**